### ISSUE TITLE
Probes are hard coded for ATLAS

### ISSUE THREAD

**Issue by ericvaandering** [rucio/probes#14]
@dchristidis CMS would like to use some of the ATLAS probes and I've verified that at least one of them works perfectly for us except for this line:

https://github.com/rucio/probes/blob/dfff3ce8a28f1157b1d9e97d5dcf7f5165a86680/common/check_transfer_queues_status#L39

Can these probes be parameterized so that we can either supply our own string for "atlas_rucio" or leave it off entirely? (Leaving it off works for us)

If you just want to parameterize one of them and leave the rest for us as we adopt them, that's OK. Whatever is easiest for you.

**Comment by dchristidis** [comment]
What does this return in your setup?
```
>>> from rucio.db.sqla.session import BASE
>>> BASE.metadata.schema
'atlas_rucio'
```

**Comment by ericvaandering** [comment]
'''
>>> from rucio.db.sqla.session import BASE
>>> BASE.metadata.schema
>>> type(BASE.metadata.schema)
<type 'NoneType'>
'''

**Comment by dchristidis** [comment]
Yes, we can definitely have a common probe. Let me use this chance to prepare some additional improvements.

**Comment by dchristidis** [comment]
Fixed by #15. Thanks!
